util/guard1/guard1.repl

```diff
@@ -0,0 +1,173 @@
+(env-data
```

Comment:
Let's add some GitHub Actions CI to ensure that these repl scripts are being run.

Reply:
CI is added